packages/service-clients/azure-client/api-report/azure-client.api.md

```diff
@@ -67,7 +67,7 @@ export interface AzureContainerVersion {
 
 // @internal @deprecated
 export class AzureFunctionTokenProvider implements ITokenProvider {
-    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "name" | "id" | "additionalDetails"> | undefined);
+    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "name" | "additionalDetails" | "id"> | undefined);
```

Comment:
Revert this one? One of those weird build issues that we see but aren't related to this PR.